app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Dropdown, Avatar, Typography } from "neetoui";
+
+const renderNameRoleAvatar = (name, { role }) => (
+  <div className="flex space-x-4">
+    <Avatar
+      user={{
+        imageUrl: "https://i.pravatar.cc/300",
+      }}
+    />
+    <div>
+      <Typography style="h5">{name}</Typography>
+      <Typography className="neeto-ui-text-gray-600" style="body3">
+        {role}
+      </Typography>
+    </div>
+  </div>
+);
+
+const renderDropdown = () => (
+  <Dropdown buttonStyle="text" icon={MenuHorizontal} strategy="fixed">
+    <li>Edit</li>
+    <li>Delete</li>
+  </Dropdown>
+);
+
+export const contactListData = [
```

Comment:
Unless there are is dynamic data being generated, this should be named as a literal constant.

Reply:
I will rectify it.